dexc/core.go

```diff
@@ -47,8 +47,12 @@ func (dc *DEXClient) InitWithPassword(pw []byte, seed []byte) error {
 	return dc.InitializeClient(pw, seed)
 }
 
-func (dc *DEXClient) IsDEXPasswordSet() bool {
-	return dc.IsInitialized()
+func (dc *DEXClient) IsInitialized() bool {
+	return dc != nil && dc.Core != nil
+}
+
+func (dc *DEXClient) InitializedWithPassword() bool {
+	return dc.Core.IsInitialized()
```

Comment:
```suggestion
	return dc. IsInitialized() && dc.Core.IsInitialized()
```

Reply:
Also, there was a `IsDEXPasswordSet` method. Removed?

Reply:
> Also, there was a IsDEXPasswordSet method. Removed?

Yes, it's the same thing.